### PR TITLE
Feature a

### PR DESCRIPTION
**Pull request by johnie261**
### Hello Microverseinc project

I have finished the Hello Microverseinc project as required. I have applied the features as following:
- Follow GitHub flow.
- Added a .gitignore file.
- Add node_modules to a .gitignore file.
- Set up a linter for HTML and CSS.
- Added a descriptive README.md.
- Added 3 files an HTML file, a CSS file, and a test.md file each with a separate commit. 
- Added test.md to the .gitignore file.

### CODE REVIEW

**Review by iambenkis — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @johnie261  👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project, but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:



 ## Required Changes ♻️

- [ ] Good job @johnie261  :+1:  setting everything up, but your PR summary should be more description as required . :100: 

Screenshot from 2023-01-11 03-13-47


You can look at this example [here](https://github.com/iambenkis/Space-X-/pull/32) and Kindly update yours!

- [ ] You are almost there with the great work :+1: , but as require you should document your README file using this [template](https://github.com/microverseinc/readme-template). In the documentation you put important information about your project. Kindly follow this template and [update](https://github.com/microverseinc/readme-template) your README file :100: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -25,11 +25,14 @@ OPTIONAL SECTIONS:
...
+=======
 <div align="center">
   <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
+>>>>>>> main
   <h3><b>Microverse README Template</b></h3>
 
 </div>
```
Comment (Shedrack-Sunday) [inline README.md]: Good job adding the README as requested by the previous reviewer. how ever, it seems you left your merge indicators on the file and kinda made it rough. please delete everything I pointed out there including the logo. The essence]of this is to give your project a professional look and let the header of your README look as shown below:
image

File: README.md [hunk 2]
```diff
@@ -40,6 +43,22 @@ After you're finished please remove all the comments and instructions!
...
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverseinc] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Hello Microverseinc]** is a...
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] The line `46` is a product of a merge issue. Kindly omit it.
- [ ] Please customize line `56` with the description of your project. the text there is to give you a guideline of what to input.

File: README.md [hunk 3]
```diff
@@ -205,6 +242,13 @@ Example:
...
+<<<<<<< HEAD
+self project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+=======
 👤 **Author1**
 
 - GitHub: [@githubhandle](https://github.com/githubhandle)
```
Comment (Shedrack-Sunday) [inline README.md]: Your social handles are not edited with your details. Kindly edit them to personalise your work 👏

File: README.md [hunk 4]
```diff
@@ -121,6 +152,11 @@ After you're finished please remove all the comments and instructions!
 
 > Describe how a new developer could make use of your project.
 
+<<<<<<< HEAD
+run index.html file
+
+
+=======
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Kindly edit the section on line `153`, as it serves as just a guide to you. You're meant to show how your project should be cloned and used.

- [ ] Once again, delete the line `155` and `159` as they are products of a merge

File: README.md [hunk 5]
```diff
@@ -92,16 +114,21 @@ After you're finished please remove all the comments and instructions!
...
 
 > Describe between 1-3 key features of the application.
 
+<<<<<<< HEAD
+- **[A simple project]**
+=======
 - **[key_feature_1]**
 - **[key_feature_2]**
 - **[key_feature_3]**
+>>>>>>> main
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] First, remove the merge indicators on lines `125, 127 and 131`.
- [ ] Secondly, please add three features your project includes

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @johnie261   , 

You did a good job so far! 👏 
However, there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Good commit messages
✅ No linter errors
✅ Good PR summary


## Required Changes ♻️
- [x] Please make sure to fill up the readme template with all the features provided.  I've listed the items omitted. It will make your work look  professional and well  documented
- [ ] After your merge, the HEAD indicators are still present in your README. Do check the comments for a detailed review

- [ ] All sections with the `>` sign are guidelines for you. you should please edit it appropriately 

- [x] Check below to see a more detailed review of what is needed. good job 👏 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @johnie261 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

✅ Good commit messages
✅ No linter errors
✅ Good PR summary

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**